update.py

```diff
@@ -27,44 +27,58 @@
 
 # Go over the release_url_map in reverse order; if a release is not yet a Git
 # tag, download the file, commit and add a tag
-did_update = False
 for tag, url, body in release_url_map[::-1]:
-  if not tag in tags:
-    print("Downloading release " + tag + " from " + url)
-    os.makedirs('./include', mode=0o777, exist_ok=True)
-    os.makedirs('./include/nlohmann', mode=0o777, exist_ok=True)
-    data = urllib.request.urlopen(url).read();
-    with open('./include/nlohmann/json.hpp', 'wb') as f:
-      f.write(data)
+  if tag in tags:
+    continue
 
-    # Try to download the json_fwd.hpp header -- only exists since release
-    # v3.1.0
-    has_json_fwd = False
-    try:
-      json_fwd_url = 'https://github.com/nlohmann/json/raw/{}/include/nlohmann/json_fwd.hpp'.format(tag);
-      print("Trying to download " + json_fwd_url)
-      data = urllib.request.urlopen(json_fwd_url).read();
-      with open('./include/nlohmann/json_fwd.hpp', 'wb') as f:
+  print("Downloading release " + tag)
+  if tag >= "v3.10.5":
+      os.system("rm -rf ./tmp")
+      os.system("git clone https://github.com/nlohmann/json ./tmp --depth 1 --branch {}".format(tag));
+      os.system("cp -rf ./tmp/single_include .")
+      os.system("cp -rf ./tmp/include/nlohmann/json_fwd.hpp ./single_include/nlohmann/")
+      os.system("cp -rf ./tmp/cmake .")
```

Comment:
Whoops nevermind, I didn't see that you copied the cmake directory. Then I guess the JSON_CI option is supported?

Reply:
JSON_CI does a lot of things. It is probably better not to support it I guess? I can't say if everything can work correctly if we don't provide the "include" directory. We only have the "single_include" one.

Reply:
True. Best to keep things minimal! And if you want to use the JSON_CI options I think you should be using the upstream library anyways and not a lightweight mirror.